opcua/common/manage_nodes.py

```diff
@@ -6,22 +6,22 @@
 
 
 def _parse_add_args(*args):
-    nodeid = ua.NodeId()
-    qname = ua.QualifiedName()
-    if isinstance(args[0], ua.NodeId):
-        nodeid = args[0]
-    elif isinstance(args[0], str):
-        nodeid = ua.NodeId.from_string(args[0])
-
-    if isinstance(args[1], ua.QualifiedName):
-        qname = args[1]
-    elif isinstance(args[1], str):
-        if isinstance(args[0], int):
+    try:
+        if isinstance(args[0], ua.NodeId):
+            nodeid = args[0]
+            qname = args[1]
```

Comment:
no we do not since it will be catched by the except and an explicative error message will be thrown